rootio/tchain.go

```diff
@@ -0,0 +1,115 @@
+package rootio
```

Comment:
this needs a license header.

see `rootio.go` for an example. (change the year to the current one, 2018)